positive_charge/src/components/seePOI/seePOI.jsx

```diff
@@ -4,23 +4,65 @@ import AddPOI from '../addPOI/AddPOI.jsx'
 import Login from '../auth/Login.jsx';
 import Signup from '../auth/Signup.jsx';
 import Modal from '../poiDetails/Modal.jsx';
+import LittleFilter from '../filter/LittleFilter.jsx';
+import BigFilter from '../filter/BigFilter.jsx';
+import Map from '../map/map.jsx';
+const axios = require('axios');
+
+
 
 class SeePOI extends React.Component {
   constructor(props) {
     super(props)
-    this.state = {};
+    this.state = {
+      lat: 39.595244,
+      long: -104.7049212,
+      dist: 600
+    };
+    this.getPOIData = this.getPOIData.bind(this);
+    this.filterForMap = this.filterForMap.bind(this);
+    this.tester = this.tester.bind(this);
+  }
+
+  getPOIData (lat, long, dist) {
+    axios.post('/getPOI', {data:{lat: this.state.lat, long: this.state.long, dist: this.state.dist}})
+    .then(result => {
+      this.setState({data: JSON.parse(result.data.body)}, () => {
+        this.filterForMap();
+      })
+    })
+  }
+
+  filterForMap () {
+    let modifiedData = [];
+    for(let i = 0; i < this.state.data.businesses.length; i++) {
+      let mapRequiredData = {
+        lat: this.state.data.businesses[i].coordinates.latitude,
+        long: this.state.data.businesses[i].coordinates.longitude,
+        destinationName: this.state.data.businesses[i].name
+      }
+      modifiedData.push(mapRequiredData);
+    }
+    this.setState({mapData: modifiedData})
   }
 
+  componentDidMount () {
+    this.getPOIData(this.state.lat, this.state.long, this.state.dist);
+
+  }
+
+  tester (){console.log('tester button', this.state)};
```

Comment:
I would suggest deleting the tester function just so no bugs show up in the future. Then it's good to merge